docs/units-and-global-variables.rst

```diff
@@ -2,23 +2,23 @@
 Units and Globally Available Variables
 **************************************
 
-.. index:: wei, finney, szabo, ether
+.. index:: wei, finney, szabo, gwei, ether
```

Comment:
```suggestion
.. index:: wei, gwei, ether
```
?

Reply:
Ah do you want to keep them indexed for the 0.7.0 note below?

Reply:
Yep... I did remove them, until @axic asked for the notice - and if we have the notice I'd say it makes sense to keep them indexed...

Reply:
Ok